docker-compose.dev.yml

```diff
@@ -3,12 +3,17 @@ version: '3'
 services:
   meican:
     image: meican-app-dev
+    env_file:
+      - .env
```

Comment:
I believe it doesn't need that part as it already exists in the docker-compose.yml file

Reply:
Removed accordingly on commit 4f6684cd